site/_guides/tls.md

```diff
@@ -17,6 +17,10 @@ Enabling TLS support requires Contour version 0.3 or later. You must also add an
 
 If you deploy behind an AWS Elastic Load Balancer, see [EC2 ELB PROXY protocol support][2] for special instructions.
 
+## TLS SNI name matching
+Envoy SNI name matching during TLS handshake is case-sensitive. For cert with common name foo.bar.com, requests to Foo.bar.com would like. Similarly, for cert with wildcard name \*.bar.com, only requests to lower case name are allowed. Here is the [known issue][4] reported on Envoy.
```

Comment:
It's better to split long lines like this up into one-sentence-per-line.

```suggestion
Envoy SNI name matching during TLS handshake is case-sensitive.
For example, for a cert with common name foo.bar.com, requests to Foo.bar.com would not match.
Similarly, for a cert with wildcard name \*.bar.com, only requests to the lower case name will match.
Here is the [known issue][4] reported on Envoy.
```

Reply:
Good point.